src/CarbonAware.DataSources/CarbonAware.DataSources.ElectricityMaps/test/Configuration/ServiceCollectionExtensionTests.cs

```diff
@@ -35,5 +38,61 @@ public void ClientProxyTest_With_Missing_ProxyURL_ThrowsException()
 
         // Act & Assert
         Assert.Throws<ConfigurationException>(() => serviceCollection.AddElectricityMapsForecastDataSource(configuration.DataSources()));
+        Assert.Throws<ConfigurationException>(() => serviceCollection.AddElectricityMapsEmissionsDataSource(configuration.DataSources()));
+    }
+
+    [Test]
```

Comment:
Suggestion: `ClientProxyTest_With_No_ProxyURL_AddsServices ` and `ClientProxyTest_With_ProxyURL_AddsServices` seem like they can be consolidated into one `TestCase` scenarios, since the code looks pretty similar. Passing a boolean flag can help to create a single test method with 2 test cases.